File: Exer2API/Exer2API/Entity/Contato.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Exer2API.Entity
{
  public class Contato
  {
    [Key]
    public int ContatoId { get; set; }

    [StringLength(1)]
    public string Tipo { get; set; }

    [StringLength(256)]
    public string Valor { get; set; }
    public int? PessoaId { get; set; }
    [ForeignKey("PessoaId")]
    public virtual Pessoa Pessoa { get; set; }
  }
}

File: BraviFront/src/app/model/Contato.ts
import { eTipo } from "./eTipo";

export class Contato {
    ContatoId : number;
    Tipo : eTipo;
    Valor: string;

    fromData(data: any){
        this.ContatoId = data.contatoId;
        this.Tipo = data.tipo;
        this.Valor = data.valor;
    }
}